test/libsolidity/syntaxTests/conversion/convert_to_super_empty.sol

```diff
@@ -0,0 +1,7 @@
+contract C {
+    function f() public pure {
+        super().x;
```

Comment:
Shouldn't this rather be an error about super not existing for the contract?

Reply:
that's a different matter